io.itch.nordup.TheGates.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.itch.nordup.TheGates
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: TheGates
+tags:
+  - proprietary
+cleanup:
+  - '*.cmake'
```

Comment:
I'm pretty there is more to clean like `/include` and maybe `*.a`

Reply:
I've also found a debug directory that serves no purpose in the build, it says it was deleted when building but the build directory still has it, does this mean it's not actually being deleted?